src/settingslist.h

```diff
@@ -379,6 +379,17 @@ SETTING_BOOL(onboardingCompleted,        // getter
              false                       // sensitive (do not log)
 )
 
+SETTING_BOOL(onboardingDataCollectionEnabled,        // getter
+             setOnboardingDataCollectionEnabled,     // setter
+             removeOnboardingDataCollectionEnabled,  // remover
+             hasOnboardingDataCollectionEnabled,     // has
+             "onboardingDataCollectionEnabled",      // key
+             false,                                  // default value
+             false,                                  // user setting
+             false,                                  // remove when reset
```

Comment:
This should be `true`, no?

Reply:
we don't need to remove this setting on reset since it gets set back to it's default setting